### PR TITLE
IA-3914 Remove googleoauth from swagger

### PR DESCRIPTION
**Pull request by Qi77Qi**
*  We've rolled b2c to prod, hence removing googleoauth from swagger. googleoauth won't work in BEEs anyway
* Bump a few dependencies

With this change, there should be only b2c login for swagger.

Screen Shot 2022-12-20 at 4 07 42 PM

---

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by mlilynolting** [comment]
Just for my curiosity, is there any context for this change? Did Google OAuth not do anything, or was it added by accident or something?

**Review by mlilynolting — APPROVED** [review]

**Comment by Qi77Qi** [comment]
@mlilynolting  it was the right auth provider before. But we've rolled out b2c that supports both google and Azure now. And the googleoauth won't work in BEE

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
fiab-start failure..jenkins retest

**Comment by Qi77Qi** [comment]
```
org.broadinstitute.dsde.workbench.leonardo.runtimes.RuntimeAutopauseSpec.autopause should work

 java.lang.NoSuchFieldError: Companion
      at okhttp3.internal.Util.<clinit>(Util.kt:70)
      at okhttp3.internal.concurrent.TaskRunner.<clinit>(TaskRunner.kt:309)
      at okhttp3.ConnectionPool.<init>(ConnectionPool.kt:41)
      at okhttp3.ConnectionPool.<init>(ConnectionPool.kt:47)
      at org.openqa.selenium.remote.internal.OkHttpClient$Factory.<init>(OkHttpClient.java:116)
      at org.openqa.selenium.remote.http.HttpClient$Factory.createDefault(HttpClient.java:66)
      at org.openqa.selenium.remote.HttpCommandExecutor.<clinit>(HttpCommandExecutor.java:47)
      at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.$anonfun$tryStart$1(WebBrowserSpec.scala:178)
      at scala.util.Try$.apply(Try.scala:210)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.tryStart(WebBrowserSpec.scala:178)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.startRemoteWebdriver(WebBrowserSpec.scala:161)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.runDockerChrome(WebBrowserSpec.scala:146)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.withWebDriver(WebBrowserSpec.scala:64)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.withWebDriver$(WebBrowserSpec.scala:60)
      at org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookGCECustomizationSpec.withWebDriver(NotebookGCECustomizationSpec.scala:22)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.withWebDriver(WebBrowserSpec.scala:50)
      at org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec.withWebDriver$(WebBrowserSpec.scala:49)
      at org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookGCECustomizationSpec.withWebDriver(NotebookGCECustomizationSpec.scala:22)
      at org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookGCECustomizationSpec.$anonfun$new$8(NotebookGCECustomizationSpec.scala:78)
      at org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookGCECustomizationSpec.$anonfun$new$8$adapted(NotebookGCECustomizationSpec.scala:77)
      at org.broadinstitute.dsde.workbench.leonardo.LeonardoTestUtils.$anonfun$withNewRuntime$1(LeonardoTestUtils.scala:499)
      at cats.effect.IOFiber.runLoop(IOFiber.scala:260)
      at cats.effect.IOFiber.execR(IOFiber.scala:1316)
      at cats.effect.IOFiber.run(IOFiber.scala:118)
      at cats.effect.unsafe.WorkerThread.run(WorkerThread.scala:555)
```

jenkins retest

**Comment by Qi77Qi** [comment]
Jenkins retest

**Comment by Qi77Qi** [comment]
stuck in leonardo-build for 22hrs

jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
fiab-start failure

jenkins retest

**Comment by Qi77Qi** [comment]
fiab-start failure

jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
jenkins retest

**Comment by Qi77Qi** [comment]
fiab-start failure `16:15:58 cromiam: timed out waiting for healthy (Degraded: cromiam-deployment: Startup probe failed: HTTP probe failed with statuscode: 503)`


jenkins retest

File: project/Dependencies.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ object Dependencies {
   val monocleV = "2.1.0"
   val opencensusV = "0.29.0"
 
-  private val workbenchLibsHash = "1a6839f"
+  private val workbenchLibsHash = "6392bb3"
```
Comment (rtitle) [inline project/Dependencies.scala]: Dependency updates seem unrelated to swagger -- are they still current?

**Review by rtitle — APPROVED** [review]
Looks good overall -- now that B2C is in prod, it's a good thing to remove the legacy google oauth from our Swagger UIs.

Guessing this PR needs a rebase, and maybe recheck the dependency updates..

**Comment by jdcanas** [comment]
jenkins retest - fiab-start

File: project/Dependencies.scala (hunk 1 above)
Comment (LizBaldo) [reply project/Dependencies.scala]: These are actually more current than what is on develop right now, looks like some bonus updates as part of this PR, might as well take it :)

**Comment by LizBaldo** [comment]
jenkins retest

**Comment by LizBaldo** [comment]
jenkin retest

**Comment by LizBaldo** [comment]
jenkins retest

File: project/Dependencies.scala (hunk 1 above)
Comment (LizBaldo) [reply project/Dependencies.scala]: Weird, the integration tests keep timing out so something might be wrong with these dependency updates in the end. I will remove that change from this PR so we can merge the swagger improvement
